src/scripts/wallet/Amount.ts

```diff
@@ -1,14 +0,0 @@
-class Amount {
-    amount: number;
-    currency: GameConstants.Currency;
-
-
-    constructor(amount: number, currency: GameConstants.Currency) {
-        this.amount = amount;
-        this.currency = currency;
-    }
-
-    public toString() {
```

Comment:
It doesn't look like this got moved over to the module?
Is this not used anywhere anymore.

Reply:
Nope, it was moved to a module just rewritten in a fancy was jessica showed me. 
![image](https://user-images.githubusercontent.com/49108377/103146763-ae9d8e80-4702-11eb-84fd-7f24517d9227.png) this part

Reply:
Ah, you will still need the `toString()` method, it was just the constructor and declarations she was talking about:
```diff
-    amount: number;
-    currency: Currency;
-
-    constructor(amount: number, currency: Currency) {
-        this.amount = amount;
-        this.currency = currency;
-    }
+    constructor(
+        public amount: number,
+        public currency: Currency
+    ) {}
```

Reply:
I ignored any methods in the class, my example was just referring to the constructor parameters which are set to some property of the class without modification. You won't get any methods like this toString() for free.

Reply:
This is too fancy for me. I'm still pretty new to this. :/ If you want to teach me something new you have to teach me the whole thing or ill learn it half-assed. :/ so how do i fix it?

Reply:
You'll just need to re-add the `toString()` method to the class

Reply:
So just readd what I deleted?

Reply:
Just the `toString()` method

Reply:
I updated it. hopefully i got it right.